modules/translation/translation.go

```diff
@@ -99,3 +100,66 @@ func (l *locale) Language() string {
 func (l *locale) Tr(format string, args ...interface{}) string {
 	return i18n.Tr(l.Lang, format, args...)
 }
+
+// Language specific rules for translating plural texts
+var trNLangRules = map[string]func(int64) int{
```

Comment:
Off-Topic:

I didn't knew about this one, is this documented somewhere for translators?

Reply:
I have no idea about `is this documented somewhere for translators`.

IMO it's mostly a knowledge for developers. If a developer write "text_1" and "text_n", the translators could just translate them without knowing `TrN`